msal-python-conceptual/advanced/msal-logging-python.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Logging errors and exceptions in MSAL for Python
+description: Learn how to log errors and exceptions in MSAL for Python
+---
+
+# Logging in MSAL for Python
+
+The Microsoft Authentication Library (MSAL) apps generate log messages that can help diagnose issues. An app can configure logging with a few lines of code, and have custom control over the level of detail and whether or not personal and organizational data is logged. We recommend you create an MSAL logging implementation and provide a way for users to submit logs when they have authentication issues.
+
+## Logging levels
+
+MSAL provides several levels of logging detail:
+
+- LogAlways: No level filtering is done on this log level. Log messages of all levels will be logged.
```

Comment:
This section (logging levels) seems to be duplicated from the previous article. Maybe we need to only reference it once and then refer to it from other articles?

Reply:
So it does! I recall there being include files. I'll keep it in this logging file (with your previous recommendations) and remove it from the other.